origin.html

```diff
@@ -16,80 +16,189 @@
+            <div class="resourcesection-column2">
+                <ul class="source">
+                    <li class="line"><img class="lineimage" src="./images/resources/Rectangle56.png" alt="">Frameworks</li>
+                    <li class="subline">Bootstrap</li>
+                    <li class="subline">Ruby on Rails</li>
+                    <li class="subline">RSPec</li>
+                    <li class="subline">CapyBara</li>
+                    <li class="subline">Selenium</li>
+                </ul>    
+            </div>
+            <div class="resourcesection-column3">
+                <ul class="source">
+                    <li class="line"><img class="lineimage" src="./images/resources/Rectangle57.png" alt="">Skills</li>
+                    <li class="subline">Codekit</li>
+                    <li class="subline">Github</li>
+                    <li class="subline">Codepen</li>
+                    <li class="subline">Gitlab</li>
+                    <li class="subline">Terminal</li>
+                </ul>
+            </div>
+        </div>
+            
+        <div class="informationsection">
+            <div class="informationsection-columns">
+                <div class="informationsection-column1">
+                    <h2 class="informationtitle">I'm always interested in hearing about new proijects, so if you'd like to chant plese get in</h2>
+                </div>
+                <div class="informationsection-column2">
+                    <form action="https://formspree.io/f/mbjwogoo" method="post">
+                        <input type="text" placeholder="Full Name" maxlength="30" required>
+                        <p class="undeline2"><img class="underline2" src="./images/information/Container.png" alt=""></p>
+                        <input type="email" placeholder="Email address" required>
+                        <p class="undeline2"><img class="underline2" src="./images/information/Container.png" alt=""></p>
+                        <textarea class="textareainformation" name="enter-text-here" maxlength="500" rows="5" placeholder="Add your text" required></textarea>
+                        <button type="submit" class="informationbutton">Get in touch</button>
+                    </form>
+                </div>
+            </div>
+            <img class="gray-line" src="./images/Rectangle.png" alt="gray_line">
+            <ul class="socialmedia2">
+                <li><a href=""><img src="./images/github.svg" alt="githublogo"></a></li>
+                <li><a href=""><img src="./images/Linkedinicon.svg" alt=""></a></li>
+                <li><a href=""><img src="./images/angellist.svg" alt="angellist"></a></li>
+                <li><a href=""><img src="./images/twitter.svg" alt="twitterlogo"></a></li>
+                <li><a href=""><img src="./images/medium.svg" alt="mediumlogo"></a></li>
             </ul>
+            <img class="black-line" src="./images/Shape.png" alt="black_line">
         </div>
```

Comment:
- [ ] The same thing for the about and contact section. you must stick to the design as much as possible (margins top, bottom, right, and left, font and font sizes) using the Figma Template provided.